src/components/ContentPreview/ContentPreview.js

```diff
@@ -625,10 +643,12 @@ class ContentPreview extends PureComponent<Props, State> {
         }
 
         const previewOptions = {
+            container: `#${this.id} .bcpr-content`,
+            header: 'none',
+            headerElement: `#${this.id} .bcpr-header`,
```

Comment:
Is it possible to refactor preview to not take / muck with DOM?
Can we just fix this by passing in functions, for example onAnnotationEnd() or something that preview annotations can trigger to toggle the button.

Reply:
I think we can add that onto a later refactor of BoxAnnotations